scripts/ci/pack_soljson.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SCRIPT_DIR="$(realpath "$(dirname "$0")")"
+SOLJSON_JS="$1"
+SOLJSON_WASM="$2"
+SOLJSON_WASM_SIZE=$(wc -c "${SOLJSON_WASM}" | cut -d ' ' -f 1)
+OUTPUT="$3"
+
+{
+  echo -n "\"use strict\";"
+  echo -n "var Module = Module || {}; Module[\"wasmBinary\"] = "
+  cat "${SCRIPT_DIR}/wasm_unpack.js"
+  echo -n "(\""
+  lz4c --no-frame-crc --best --favor-decSpeed "${SOLJSON_WASM}" - | base64 -w 0 | sed 's/[^A-Za-z0-9\+\/=]//g'
```

Comment:
https://developer.mozilla.org/en-US/docs/Glossary/Base64 had an explicit ``sBase64.replace(/[^A-Za-z0-9\+\/]/g, "")`` (not that this does not have a ``=``), but since we're in full control of the input, I moved it here instead, s.t. we don't need to do it on runtime.
But since the current version uses ``Buffer.from`` if available, I then added the ``=``, s.t. the string is proper valid base64 - which makes the ``sed`` superfluous, since ``base64 -w0`` should indeed produce just that, but it also doesn't hurt.
Having the additional ``=`` should be robust with still forgoing the ``sBase64.replace(/[^A-Za-z0-9\+\/]/g, "")``, if I see it correctly, since the decoding routine will just stop at ``compSize === 0`` anyways.

So, depends - if we stay with using ``Buffer.from``, we might as well remove the ``sed`` - otherwise I'll add it back to remove the ``=`` at the end.